custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py

```diff
@@ -571,8 +571,16 @@ def get_case_structures_from_row(commit, domain, migration_id, column_mapping, c
     )
 
     # Close the occurrence if we have a treatment outcome recorded
-    close_occurrence = ("treatment_outcome" in episode_case_properties and
-                        episode_case_properties["treatment_outcome"])
+    if (
+        "treatment_outcome" in episode_case_properties
+        and episode_case_properties["treatment_outcome"]
+    ):
+        close_occurrence = True
```

Comment:
do you need to add a close reason or something?. Also, do you need to close the episode?

Reply:
The app doesn't add a close reason to the occurrence for this.  

I was assuming that since episode is an extension, it would get auto-closed as well.  Is that not true?

Reply:
it looks like you are opening these cases at the same time as closing them, is that true? 

If so, you might be creating the closed occurrence before you create the episode case, in which case the episode case will remain open.

Reply:
Ok, I'll manually go through and close out the other cases then too.